lib/__tests__/react-loading.spec.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { shallow } from 'enzyme';
+import sinon from 'sinon';
```

Comment:
Do we need `sinon` since we have `jest`? Not sure but i think that this might work:

```javascript
jest.spyOn(global.console, 'warn')
```